neo/test/rawiotest/test_medrawio.py

```diff
@@ -0,0 +1,241 @@
+        
+        # Verify it found all 3 channels.
+        self.assertEqual(rawio.num_channels_in_session, 3)
+        self.assertEqual(rawio.header['signal_channels'].size, 3)
+  
+        # Verity if found the names of the 3 channels
+        self.assertEqual(rawio.header['signal_channels'][0][0], '5k_ch1')
+        self.assertEqual(rawio.header['signal_channels'][1][0], '1k_ch1')
+        self.assertEqual(rawio.header['signal_channels'][2][0], '1k_ch2')
+        
+        # Read first 3 seconds of data from the first channel (5k_ch1)
+        raw_chunk = rawio.get_analogsignal_chunk(block_index=0,
+                                seg_index=0,
+                                i_start=0,
+                                i_stop=15000,
+                                stream_index=0,
+                                channel_indexes=None)
+                                
+        # Test the first three sample values returned, which are
+        # known to be [-80, -79, -78]
+        self.assertEqual(raw_chunk[0][0], -80)
+        self.assertEqual(raw_chunk[1][0], -79)
+        self.assertEqual(raw_chunk[2][0], -78)
+   
+        
+        # Read first 3 seconds of data from the second channel and third
+        # channels (1k_ch1 and 1k_ch2)
+        raw_chunk = rawio.get_analogsignal_chunk(block_index=0,
+                                seg_index=0,
+                                i_start=0,
+                                i_stop=3000,
+                                stream_index=1,
+                                channel_indexes=None)
+               
+        # Test first sample returned of both channels, which are known
+        # to be [-79, -80]
+        np.testing.assert_array_equal(raw_chunk[0][:2], [-79, -80])
+        
+        # Read first 3 seconds of data from the second segment of the first channel (5k_ch1)
+        raw_chunk = rawio.get_analogsignal_chunk(block_index=0,
+                                seg_index=1,
+                                i_start=0,
+                                i_stop=15000,
+                                stream_index=0,
+                                channel_indexes=None)
+                                
+        # Test the first three sample values returned, which are
+        # known to be [22, 23, 24]
+        self.assertEqual(raw_chunk[0][0], 22)
+        self.assertEqual(raw_chunk[1][0], 23)
+        self.assertEqual(raw_chunk[2][0], 24)
+                                
+        self.assertEqual(len(rawio.header['event_channels']), 2)
+        
+        # Verify that there are 5 events in the dataset.
+        # They are 3 "Note" type events, and 2 "NlxP", or neuralynx, type events.
+        # The first segment has one event, and the second and third segments
+        # each have 2 events.
+        self.assertEqual(rawio._event_count(0, 0, 0), 1)
+        self.assertEqual(rawio._event_count(0, 1, 0), 2)
+        self.assertEqual(rawio._event_count(0, 2, 0), 2)
```

Comment:
You can also use the exposed event count methods directly
```suggestion
        self.assertEqual(rawio.event_count(0, 0, 0), 1)
        self.assertEqual(rawio.event_count(0, 1, 0), 2)
        self.assertEqual(rawio.event_count(0, 2, 0), 2)
```